pkg/services/accesscontrol/database/database.go

```diff
@@ -241,3 +241,58 @@ func (s *AccessControlStore) DeleteUserPermissions(ctx context.Context, orgID, u
 	})
 	return err
 }
+
+func (s *AccessControlStore) DeleteTeamPermissions(ctx context.Context, orgID, teamID int64) error {
```

Comment:
Note that the orgID is not really mandatory, but it provides some consistency with `DeleteUserPermissions`